### ISSUE TITLE
retrieve the model of a part instance

### ISSUE THREAD

**Issue by jberends** [KE-works/pykechain#77]
in the Part class
```
def model():
   # add check if instance, otherwise nothing
   the_model = get(self._json_data, 'model')
   return # the part that is the model (so do a retrieve to KEC)
```

- [ ] make a new branch from master called #77-model-of-part
- [ ] write code in part
- [ ] write tests
- [ ] do intermediate commits and push to github
- [ ] ensure a merge request is made from your branch

**Comment by thendrich** [comment]
@jberends cc @wilcoschoneveld 

+1, I'm having the same requests.

Things I'd like to do:

- Retrieve the model of an instance
- Retrieve the instances of a model
- Retrieve the proxies of a catalog model
- Retrieve the catalog model of proxy 
- Retrieve the (direct) properties of a part instance
- Retrieve the (direct) properties of a part model

This way, I can lookup sets of parts/properties and act on them (e.g. configure them). Currently, without the above, it's difficult to do more complex operations since retrieving the right parts is difficult.

**Comment by thendrich** [comment]
Made a new issue #79

**Comment by jberends** [comment]
@thendrich good that you made a new ticket
This ticket is included in 1.6.0 to be released
